nidirect_gp/js/gp-location-search.js

```diff
@@ -18,8 +24,16 @@
       }
 
       // Callback function for geolocation errors.
-      function locationError() {
-        const confirmationDialog = Drupal.dialog('<div>Unable to determine your location.</div>', {
+      function locationError(err) {
+
+        console.warn(`ERROR(${err.code}): ${err.message}`);
+        $('#find-by-location-status').html('');
+
+        let errmsg = '' +
```

Comment:
Looking back at my code, ideally I should have passed it through Drupal's translation function and we should consider doing that here.

Reply:
Yep never thought of that - I've added it here and a couple of other spots.